### DISCUSSION TITLE
Problem using SVBony SV305

### DISCUSSION THREAD

**Discussion by enriquemarquez** [aaronwmorris/indi-allsky#899] [General]
First of all, thank you very much for everything developing this software. 

I have a problem that should be surely something small enough but I can't fix it. I read lots of threads but I can't. I compile several times indilib and proceed with indi-allsky setup. It was working with astroberry last year but I noticed that it is no longer supported so I change to raspbian. Thank you very much in advance. 

- The error is:

Aug 19 21:36:44 raspberrypi [INFO] MainProcess/MainThread allsky._queueManualTasks() #1114: Checking for manually submitted tasks
Aug 19 21:36:44 raspberrypi [INFO] MainProcess/MainThread allsky._startCaptureWorker() #286: Starting Capture-30 worker
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception: 
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception: indi_allsky.exceptions.TimeOutException: Timeout finding control FRAME_FORMAT
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     raise TimeOutException('Timeout finding control {0}'.format(name))
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/camera/indi.py", line 1221, in get_control
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     c = self.get_control(device, name, 'switch')
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/camera/indi.py", line 1249, in set_switch
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     self.set_switch(device, k, on_switches=v.get('on', []), off_switches=v.get('off', []))
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/camera/indi.py", line 698, in configureDevice
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     self.configureDevice(self._ccd_device, *args, **kwargs)
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/camera/indi.py", line 716, in configureCcdDevice
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     self.indiclient.configureCcdDevice(self.config['INDI_CONFIG_DEFAULTS'])
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/capture.py", line 577, in _initialize
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     self._initialize()
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/capture.py", line 185, in saferun
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:     self.saferun()
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception:   File "/home/eragonsmith/indi-allsky/indi_allsky/capture.py", line 173, in run
Aug 19 21:36:44 raspberrypi [ERROR] MainProcess/MainThread allsky._startCaptureWorker() #279: Capture worker exception: Traceback (most recent call last):

- The output of indi_getprop is:

SVBONY SV305 0.CONNECTION.CONNECT=On
SVBONY SV305 0.CONNECTION.DISCONNECT=Off
SVBONY SV305 0.DRIVER_INFO.DRIVER_NAME=SVBONY CCD
SVBONY SV305 0.DRIVER_INFO.DRIVER_EXEC=indi_svbony_ccd
SVBONY SV305 0.DRIVER_INFO.DRIVER_VERSION=1.3
SVBONY SV305 0.DRIVER_INFO.DRIVER_INTERFACE=2
SVBONY SV305 0.POLLING_PERIOD.PERIOD_MS=1000
SVBONY SV305 0.CONFIG_PROCESS.CONFIG_LOAD=Off
SVBONY SV305 0.CONFIG_PROCESS.CONFIG_SAVE=Off
SVBONY SV305 0.CONFIG_PROCESS.CONFIG_DEFAULT=Off
SVBONY SV305 0.CONFIG_PROCESS.CONFIG_PURGE=Off
SVBONY SV305 0.DEBUG.ENABLE=Off
SVBONY SV305 0.DEBUG.DISABLE=On
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_TELESCOPE=Telescope Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_ROTATOR=Rotator Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_FOCUSER=Focuser Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_FILTER=CCD Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_SKYQUALITY=SQM
SVBONY SV305 0.CCD_EXPOSURE.CCD_EXPOSURE_VALUE=1
SVBONY SV305 0.CCD_ABORT_EXPOSURE.ABORT=Off
SVBONY SV305 0.CCD_FRAME.X=0
SVBONY SV305 0.CCD_FRAME.Y=0
SVBONY SV305 0.CCD_FRAME.WIDTH=1920
SVBONY SV305 0.CCD_FRAME.HEIGHT=1080
SVBONY SV305 0.CCD_BINNING.HOR_BIN=1
SVBONY SV305 0.CCD_BINNING.VER_BIN=1
SVBONY SV305 0.CCD_CAPTURE_FORMAT.FORMAT_RAW8=Off
SVBONY SV305 0.CCD_CAPTURE_FORMAT.FORMAT_RAW16=On
SVBONY SV305 0.CCD_CAPTURE_FORMAT.FORMAT_Y8=Off
SVBONY SV305 0.CCD_TRANSFER_FORMAT.FORMAT_FITS=On
SVBONY SV305 0.CCD_TRANSFER_FORMAT.FORMAT_NATIVE=Off
SVBONY SV305 0.CCD_TRANSFER_FORMAT.=Off
SVBONY SV305 0.CCD_INFO.CCD_MAX_X=1920
SVBONY SV305 0.CCD_INFO.CCD_MAX_Y=1080
SVBONY SV305 0.CCD_INFO.CCD_PIXEL_SIZE=2.9000000953674316406
SVBONY SV305 0.CCD_INFO.CCD_PIXEL_SIZE_X=2.9000000953674316406
SVBONY SV305 0.CCD_INFO.CCD_PIXEL_SIZE_Y=2.9000000953674316406
SVBONY SV305 0.CCD_INFO.CCD_BITSPERPIXEL=16
SVBONY SV305 0.CCD_COMPRESSION.INDI_ENABLED=Off
SVBONY SV305 0.CCD_COMPRESSION.INDI_DISABLED=On
SVBONY SV305 0.CCD_FRAME_TYPE.FRAME_LIGHT=On
SVBONY SV305 0.CCD_FRAME_TYPE.FRAME_BIAS=Off
SVBONY SV305 0.CCD_FRAME_TYPE.FRAME_DARK=Off
SVBONY SV305 0.CCD_FRAME_TYPE.FRAME_FLAT=Off
SVBONY SV305 0.CCD_CFA.CFA_OFFSET_X=0
SVBONY SV305 0.CCD_CFA.CFA_OFFSET_Y=0
SVBONY SV305 0.CCD_CFA.CFA_TYPE=GRBG
SVBONY SV305 0.SCOPE_INFO.FOCAL_LENGTH=0
SVBONY SV305 0.SCOPE_INFO.APERTURE=0
SVBONY SV305 0.WCS_CONTROL.WCS_ENABLE=Off
SVBONY SV305 0.WCS_CONTROL.WCS_DISABLE=On
SVBONY SV305 0.UPLOAD_MODE.UPLOAD_CLIENT=On
SVBONY SV305 0.UPLOAD_MODE.UPLOAD_LOCAL=Off
SVBONY SV305 0.UPLOAD_MODE.UPLOAD_BOTH=Off
SVBONY SV305 0.UPLOAD_SETTINGS.UPLOAD_DIR=/home/eragonsmith
SVBONY SV305 0.UPLOAD_SETTINGS.UPLOAD_PREFIX=IMAGE_XXX
SVBONY SV305 0.CCD_FAST_TOGGLE.INDI_ENABLED=Off
SVBONY SV305 0.CCD_FAST_TOGGLE.INDI_DISABLED=On
SVBONY SV305 0.CCD_FAST_COUNT.FRAMES=1
SVBONY SV305 0.CCD_VIDEO_STREAM.STREAM_ON=Off
SVBONY SV305 0.CCD_VIDEO_STREAM.STREAM_OFF=On
SVBONY SV305 0.STREAM_DELAY.STREAM_DELAY_TIME=0
SVBONY SV305 0.STREAMING_EXPOSURE.STREAMING_EXPOSURE_VALUE=0.10000000000000000555
SVBONY SV305 0.STREAMING_EXPOSURE.STREAMING_DIVISOR_VALUE=1
SVBONY SV305 0.FPS.EST_FPS=30
SVBONY SV305 0.FPS.AVG_FPS=30
SVBONY SV305 0.RECORD_STREAM.RECORD_ON=Off
SVBONY SV305 0.RECORD_STREAM.RECORD_DURATION_ON=Off
SVBONY SV305 0.RECORD_STREAM.RECORD_FRAME_ON=Off
SVBONY SV305 0.RECORD_STREAM.RECORD_OFF=On
SVBONY SV305 0.RECORD_FILE.RECORD_FILE_DIR=/home/eragonsmith/indi__D_
SVBONY SV305 0.RECORD_FILE.RECORD_FILE_NAME=indi_record__T_
SVBONY SV305 0.RECORD_OPTIONS.RECORD_DURATION=1
SVBONY SV305 0.RECORD_OPTIONS.RECORD_FRAME_TOTAL=30
SVBONY SV305 0.CCD_STREAM_FRAME.X=0
SVBONY SV305 0.CCD_STREAM_FRAME.Y=0
SVBONY SV305 0.CCD_STREAM_FRAME.WIDTH=0
SVBONY SV305 0.CCD_STREAM_FRAME.HEIGHT=0
SVBONY SV305 0.CCD_STREAM_ENCODER.RAW=On
SVBONY SV305 0.CCD_STREAM_ENCODER.MJPEG=Off
SVBONY SV305 0.CCD_STREAM_RECORDER.SER=On
SVBONY SV305 0.LIMITS.LIMITS_BUFFER_MAX=512
SVBONY SV305 0.LIMITS.LIMITS_PREVIEW_FPS=10
SVBONY SV305 0.CCD_GAIN.GAIN=10
SVBONY SV305 0.CCD_CONTRAST.CONTRAST=50
SVBONY SV305 0.CCD_SHARPNESS.SHARPNESS=0
SVBONY SV305 0.CCD_SATURATION.SATURATION=128
SVBONY SV305 0.CCD_WBR.WBR=128
SVBONY SV305 0.CCD_WBG.WBG=128
SVBONY SV305 0.CCD_WBB.WBB=128
SVBONY SV305 0.CCD_GAMMA.GAMMA=100
SVBONY SV305 0.CCD_OFFSET.OFFSET=0
SVBONY SV305 0.FRAME_RATE.SPEED_SLOW=Off
SVBONY SV305 0.FRAME_RATE.SPEED_NORMAL=On
SVBONY SV305 0.FRAME_RATE.SPEED_FAST=Off
SVBONY SV305 0.STRETCH_BITS.STRETCH_OFF=On
SVBONY SV305 0.STRETCH_BITS.STRETCH_X2=Off
SVBONY SV305 0.STRETCH_BITS.STRETCH_X4=Off
SVBONY SV305 0.STRETCH_BITS.STRETCH_X8=Off
SVBONY SV305 0.STRETCH_BITS.STRETCH_X16=Off
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_TELESCOPE=Telescope Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_ROTATOR=Rotator Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_FOCUSER=Focuser Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_FILTER=CCD Simulator
SVBONY SV305 0.ACTIVE_DEVICES.ACTIVE_SKYQUALITY=SQM
SVBONY SV305 0.CCD_VIDEO_STREAM.STREAM_ON=Off
SVBONY SV305 0.CCD_VIDEO_STREAM.STREAM_OFF=On
SVBONY SV305 0.STREAMING_EXPOSURE.STREAMING_EXPOSURE_VALUE=0.10000000000000000555
SVBONY SV305 0.STREAMING_EXPOSURE.STREAMING_DIVISOR_VALUE=1
SVBONY SV305 0.FPS.EST_FPS=30
SVBONY SV305 0.FPS.AVG_FPS=30
SVBONY SV305 0.RECORD_STREAM.RECORD_ON=Off
SVBONY SV305 0.RECORD_STREAM.RECORD_DURATION_ON=Off
SVBONY SV305 0.RECORD_STREAM.RECORD_FRAME_ON=Off
SVBONY SV305 0.RECORD_STREAM.RECORD_OFF=On
SVBONY SV305 0.RECORD_FILE.RECORD_FILE_DIR=/home/eragonsmith/indi__D_
SVBONY SV305 0.RECORD_FILE.RECORD_FILE_NAME=indi_record__T_
SVBONY SV305 0.RECORD_OPTIONS.RECORD_DURATION=1
SVBONY SV305 0.RECORD_OPTIONS.RECORD_FRAME_TOTAL=30
SVBONY SV305 0.CCD_STREAM_FRAME.X=0
SVBONY SV305 0.CCD_STREAM_FRAME.Y=0
SVBONY SV305 0.CCD_STREAM_FRAME.WIDTH=0
SVBONY SV305 0.CCD_STREAM_FRAME.HEIGHT=0
SVBONY SV305 0.CCD_STREAM_ENCODER.RAW=On
SVBONY SV305 0.CCD_STREAM_ENCODER.MJPEG=Off
SVBONY SV305 0.CCD_STREAM_RECORDER.SER=On
SVBONY SV305 0.LIMITS.LIMITS_BUFFER_MAX=512
SVBONY SV305 0.LIMITS.LIMITS_PREVIEW_FPS=10
Telescope Simulator.CONNECTION.CONNECT=On
Telescope Simulator.CONNECTION.DISCONNECT=Off
Telescope Simulator.DRIVER_INFO.DRIVER_NAME=Telescope Simulator
Telescope Simulator.DRIVER_INFO.DRIVER_EXEC=indi_simulator_telescope
Telescope Simulator.DRIVER_INFO.DRIVER_VERSION=1.0
Telescope Simulator.DRIVER_INFO.DRIVER_INTERFACE=5
Telescope Simulator.POLLING_PERIOD.PERIOD_MS=250
Telescope Simulator.DEBUG.ENABLE=Off
Telescope Simulator.DEBUG.DISABLE=On
Telescope Simulator.CONFIG_PROCESS.CONFIG_LOAD=Off
Telescope Simulator.CONFIG_PROCESS.CONFIG_SAVE=Off
Telescope Simulator.CONFIG_PROCESS.CONFIG_DEFAULT=Off
Telescope Simulator.CONFIG_PROCESS.CONFIG_PURGE=Off
Telescope Simulator.CONNECTION_MODE.CONNECTION_SERIAL=On
Telescope Simulator.CONNECTION_MODE.CONNECTION_TCP=Off
Telescope Simulator.DEVICE_PORT.PORT=/dev/ttyUSB0
Telescope Simulator.DEVICE_BAUD_RATE.9600=On
Telescope Simulator.DEVICE_BAUD_RATE.19200=Off
Telescope Simulator.DEVICE_BAUD_RATE.38400=Off
Telescope Simulator.DEVICE_BAUD_RATE.57600=Off
Telescope Simulator.DEVICE_BAUD_RATE.115200=Off
Telescope Simulator.DEVICE_BAUD_RATE.230400=Off
Telescope Simulator.DEVICE_AUTO_SEARCH.INDI_ENABLED=On
Telescope Simulator.DEVICE_AUTO_SEARCH.INDI_DISABLED=Off
Telescope Simulator.DEVICE_PORT_SCAN.Scan Ports=Off
Telescope Simulator.ACTIVE_DEVICES.ACTIVE_GPS=GPS Simulator
Telescope Simulator.ACTIVE_DEVICES.ACTIVE_DOME=Dome Simulator
Telescope Simulator.DOME_POLICY.DOME_IGNORED=On
Telescope Simulator.DOME_POLICY.DOME_LOCKS=Off
Telescope Simulator.TELESCOPE_INFO.TELESCOPE_APERTURE=10
Telescope Simulator.TELESCOPE_INFO.TELESCOPE_FOCAL_LENGTH=10
Telescope Simulator.TELESCOPE_INFO.GUIDER_APERTURE=120
Telescope Simulator.TELESCOPE_INFO.GUIDER_FOCAL_LENGTH=900
Telescope Simulator.SCOPE_CONFIG_NAME.SCOPE_CONFIG_NAME=indi-allsky
Telescope Simulator.MOUNT_AXES.PRIMARY=1.3448298545881700491e-06
Telescope Simulator.MOUNT_AXES.SECONDARY=36.728580000000000894
Telescope Simulator.ON_COORD_SET.TRACK=On
Telescope Simulator.ON_COORD_SET.SLEW=Off
Telescope Simulator.ON_COORD_SET.SYNC=Off
Telescope Simulator.EQUATORIAL_EOD_COORD.RA=17.275154344002885409
Telescope Simulator.EQUATORIAL_EOD_COORD.DEC=36.72858047485350852
Telescope Simulator.TELESCOPE_ABORT_MOTION.ABORT=Off
Telescope Simulator.TELESCOPE_TRACK_MODE.TRACK_SIDEREAL=On
Telescope Simulator.TELESCOPE_TRACK_MODE.TRACK_SOLAR=Off
Telescope Simulator.TELESCOPE_TRACK_MODE.TRACK_LUNAR=Off
Telescope Simulator.TELESCOPE_TRACK_MODE.TRACK_CUSTOM=Off
Telescope Simulator.TELESCOPE_TRACK_STATE.TRACK_ON=Off
Telescope Simulator.TELESCOPE_TRACK_STATE.TRACK_OFF=On
Telescope Simulator.TELESCOPE_TRACK_RATE.TRACK_RATE_RA=15.04106717867020393
Telescope Simulator.TELESCOPE_TRACK_RATE.TRACK_RATE_DE=0
Telescope Simulator.TELESCOPE_MOTION_NS.MOTION_NORTH=Off
Telescope Simulator.TELESCOPE_MOTION_NS.MOTION_SOUTH=Off
Telescope Simulator.TELESCOPE_MOTION_WE.MOTION_WEST=Off
Telescope Simulator.TELESCOPE_MOTION_WE.MOTION_EAST=Off
Telescope Simulator.TELESCOPE_REVERSE_MOTION.REVERSE_NS=Off
Telescope Simulator.TELESCOPE_REVERSE_MOTION.REVERSE_WE=Off
Telescope Simulator.TELESCOPE_SLEW_RATE.1x=Off
Telescope Simulator.TELESCOPE_SLEW_RATE.2x=Off
Telescope Simulator.TELESCOPE_SLEW_RATE.3x=Off
Telescope Simulator.TELESCOPE_SLEW_RATE.4x=On
Telescope Simulator.TARGET_EOD_COORD.RA=0
Telescope Simulator.TARGET_EOD_COORD.DEC=0
Telescope Simulator.TIME_UTC.UTC=2023-08-19T19:24:21
Telescope Simulator.TIME_UTC.OFFSET=2.00
Telescope Simulator.GEOGRAPHIC_COORD.LAT=36.728580474853515625
Telescope Simulator.GEOGRAPHIC_COORD.LONG=-4.4068112373352050781
Telescope Simulator.GEOGRAPHIC_COORD.ELEV=0
Telescope Simulator.TELESCOPE_PARK.PARK=On
Telescope Simulator.TELESCOPE_PARK.UNPARK=Off
Telescope Simulator.TELESCOPE_PARK_POSITION.PARK_HA=0
Telescope Simulator.TELESCOPE_PARK_POSITION.PARK_DEC=36.728580474853515625
Telescope Simulator.TELESCOPE_PARK_OPTION.PARK_CURRENT=Off
Telescope Simulator.TELESCOPE_PARK_OPTION.PARK_DEFAULT=Off
Telescope Simulator.TELESCOPE_PARK_OPTION.PARK_WRITE_DATA=Off
Telescope Simulator.TELESCOPE_PARK_OPTION.PARK_PURGE_DATA=Off
Telescope Simulator.TELESCOPE_PIER_SIDE.PIER_WEST=Off
Telescope Simulator.TELESCOPE_PIER_SIDE.PIER_EAST=On
Telescope Simulator.APPLY_SCOPE_CONFIG.SCOPE_CONFIG1=On
Telescope Simulator.APPLY_SCOPE_CONFIG.SCOPE_CONFIG2=Off
Telescope Simulator.APPLY_SCOPE_CONFIG.SCOPE_CONFIG3=Off
Telescope Simulator.APPLY_SCOPE_CONFIG.SCOPE_CONFIG4=Off
Telescope Simulator.APPLY_SCOPE_CONFIG.SCOPE_CONFIG5=Off
Telescope Simulator.APPLY_SCOPE_CONFIG.SCOPE_CONFIG6=Off
Telescope Simulator.USEJOYSTICK.ENABLE=Off
Telescope Simulator.USEJOYSTICK.DISABLE=On
Telescope Simulator.SNOOP_JOYSTICK.SNOOP_JOYSTICK_DEVICE=Joystick
Telescope Simulator.TELESCOPE_TIMED_GUIDE_NS.TIMED_GUIDE_N=0
Telescope Simulator.TELESCOPE_TIMED_GUIDE_NS.TIMED_GUIDE_S=0
Telescope Simulator.TELESCOPE_TIMED_GUIDE_WE.TIMED_GUIDE_W=0
Telescope Simulator.TELESCOPE_TIMED_GUIDE_WE.TIMED_GUIDE_E=0
Telescope Simulator.GUIDE_RATE.GUIDE_RATE_WE=0.5
Telescope Simulator.GUIDE_RATE.GUIDE_RATE_NS=0.5
Telescope Simulator.ACTIVE_DEVICES.ACTIVE_GPS=GPS Simulator
Telescope Simulator.ACTIVE_DEVICES.ACTIVE_DOME=Dome Simulator
Telescope Simulator.DOME_POLICY.DOME_IGNORED=On
Telescope Simulator.DOME_POLICY.DOME_LOCKS=Off
Telescope Simulator.TELESCOPE_INFO.TELESCOPE_APERTURE=10
Telescope Simulator.TELESCOPE_INFO.TELESCOPE_FOCAL_LENGTH=10
Telescope Simulator.TELESCOPE_INFO.GUIDER_APERTURE=120
Telescope Simulator.TELESCOPE_INFO.GUIDER_FOCAL_LENGTH=900
Telescope Simulator.SCOPE_CONFIG_NAME.SCOPE_CONFIG_NAME=indi-allsky
Telescope Simulator.USEJOYSTICK.ENABLE=Off
Telescope Simulator.USEJOYSTICK.DISABLE=On
Telescope Simulator.SNOOP_JOYSTICK.SNOOP_JOYSTICK_DEVICE=Joystick
Telescope Simulator.MOUNT_AXES.PRIMARY=1.3448298545881700491e-06
Telescope Simulator.MOUNT_AXES.SECONDARY=36.728580000000000894

**Comment by aaronwmorris** [comment]
Very strange.  It is like the FRAME_FORMAT switch is no longer available.  I will hook up my SV305 and run some tests.

**Comment by aaronwmorris** [comment]
I have updated the INDI customization page for the new parameters.

```json
{
    "PROPERTIES" : {},
    "SWITCHES" : {
        "CCD_CAPTURE_FORMAT" : {
            "on"  : ["FORMAT_RAW16"],
            "off" : ["FORMAT_RAW8"]
        }
    }
}
```

**Reply by enriquemarquez** [reply]
Thank you very much Aaron for your time. It works perfectly.
